From a017046363d548ada3aebf9b64ff3717c369a9c3 Mon Sep 17 00:00:00 2001
From: Wu Zheng <wu.zheng@intel.com>
Date: Thu, 30 Jun 2016 14:37:08 +0800
Subject: [PATCH] Modify the WIFI config to enable the Runtime Power Management

Modify the WIFI config in the backport WIFI meta layer to
enable the Runtime Power Management.
---
 defconfigs/iwlwifi-public | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/defconfigs/iwlwifi-public b/defconfigs/iwlwifi-public
index 4ff4264..646f7b9 100644
--- a/defconfigs/iwlwifi-public
+++ b/defconfigs/iwlwifi-public
@@ -18,7 +18,8 @@ CPTCFG_IWLWIFI_DEVICE_TRACING=y
 CPTCFG_IWLWIFI_DEVICE_TESTMODE=y
 CPTCFG_IWLWIFI_PCIE=y
 CPTCFG_IWLXVT=m
-CPTCFG_IWLWIFI_SDIO=y
+# CPTCFG_IWLWIFI_SDIO is not set
 CPTCFG_IWLWIFI_FRQ_MGR_TEST=y
 CPTCFG_IWLWIFI_OPMODE_MODULAR=y
 CPTCFG_IWLWIFI_LTE_COEX=y
+CPTCFG_IWLWIFI_PCIE_RTPM=y
-- 
2.1.4

